src/new_addon.ts

```diff
@@ -6,22 +6,23 @@ import { letUserPickItem } from './select_utils';
 import {
     cancel, readTextFile, writeTextFile, getWorkspaceFolders,
     addFolderToWorkspace, multiReplaceText, pathExists,
-    isValidPythonModuleName, renamePath
+    isValidPythonModuleName, renamePath, toTitleCase
 } from './utils';
 
-type AddonBuilder = (path: string, addonName: string, authorName: string) => Promise<string>;
+type AddonBuilder = (path: string, addonName: string, authorName: string, supportLegacy: boolean) => Promise<string>;
 
 const addonTemplateDir = path.join(templateFilesDir, 'addons');
+const manifestFile = path.join(addonTemplateDir, '..', 'blender_manifest.toml')
 
 export async function COMMAND_newAddon() {
     let builder = await getNewAddonGenerator();
-    let [addonName, authorName] = await askUser_SettingsForNewAddon();
+    let [addonName, authorName, supportLegacy] = await askUser_SettingsForNewAddon();
     let folderPath = await getFolderForNewAddon();
     folderPath = await fixAddonFolderName(folderPath);
-    let mainPath = await builder(folderPath, addonName, authorName);
+    let mainPath = await builder(folderPath, addonName, authorName, supportLegacy);
 
     await vscode.window.showTextDocument(vscode.Uri.file(mainPath));
-    await vscode.commands.executeCommand('cursorBottom');
+    // await vscode.commands.executeCommand('cursorBottom');
```

Comment:
Why this change? If it should be changed, then the line should be removed instead.

Reply:
Since the manifest is now opened instead of the init file, moving the cursor to the bottom would hide the important info to change off the top of the screen. I just forgot to remove this line.